specification/managementgroups/resource-manager/readme.md

```diff
@@ -0,0 +1,171 @@
+  - repo: azure-libraries-for-java
+  - repo: azure-sdk-for-go
+```
+
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  namespace: Microsoft.Azure.Management.ManagementGroups
+  output-folder: $(csharp-sdks-folder)/ManagementGroups/Management.ManagementGroups/Generated
+  clear-output-folder: true
+```
+
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+Use `--python-mode=update` if you already have a setup.py and just want to update the code itself.
+
+``` yaml $(python)
+python-mode: create
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  namespace: azure.mgmt.managementgroups
+  package-name: azure-mgmt-managementgroups
+  clear-output-folder: true
+```
+``` yaml $(python) && $(python-mode) == 'update'
+python:
+  no-namespace-folders: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-managementgroups/azure/mgmt/managementgroups
+```
+``` yaml $(python) && $(python-mode) == 'create'
+python:
+  basic-setup-py: true
+  output-folder: $(python-sdks-folder)/azure-mgmt-managementgroups
+```
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+  namespace: managementgroups
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2018-01
+  - tag: package-2017-11
+  - tag: package-2017-08
+```
+
+### Tag: package-2018-01 and go
+
+These settings apply only when `--tag=package-2018-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2018-01' && $(go)
+output-folder: $(go-sdk-folder)/services/managementgroups/mgmt/2018-01-01-preview/managementgroups
+```
+
+### Tag: package-2017-11 and go
+
+These settings apply only when `--tag=package-2017-11 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2017-11' && $(go)
+output-folder: $(go-sdk-folder)/services/managementgroups/mgmt/2017-11-01-preview/managementgroups
+```
+
+### Tag: package-2017-08 and go
+
+These settings apply only when `--tag=package-2017-08 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag)=='package-2017-08' && $(go)
+output-folder: $(go-sdk-folder)/services/managementgroups/mgmt/2017-08-31-preview/managementgroups
```

Comment:
The Go SDK has already shipped these clients to the former locations, as seen here:
[Azure/azure-sdk-for-go@21b6814](https://github.com/Azure/azure-sdk-for-go/tree/21b68149ccf7c16b3f028bb4c7fd0ab458fe308f/services/resources/mgmt)

Seeing as this would cause a breaking change for us, I'd like to leave the `output-folder` portion alone. Mind changing these to their old values?

Reply:
Done. However, I can keep the new paths for the later api versions right?